src/Clide.IntegrationTests/Solution/FolderNodeFactorySpec.cs

```diff
@@ -28,10 +28,17 @@ public override void when_item_is_not_supported_then_factory_returns_false_and_c
     [Collection("OpenSolution11")]
     public class FolderNodeFactorySpec : NodeFactorySpec<FolderNodeFactory>
     {
+        private readonly OpenSolution11Fixture fixture;
+
+        public FolderNodeFactorySpec(OpenSolution11Fixture fixture)
+        {
+            this.fixture = fixture;
+        }
+
         [InlineData("Native\\CsLibrary\\CsFolder")]
         [InlineData("Native\\VbLibrary\\VbFolder")]
         [InlineData("Native\\FsLibrary\\FsFolder")]
-        [VsTheory]
+        [VsTheory(ReuseInstance = false)]
```

Comment:
❔ This will cause a _serious_ performance hit to test runs. Are you sure it is necessary?

Reply:
yes, we're aware of that. It should not be necessary we also want to avoid this as much as possible. I need to investigate why they fail when all tests are executed together. But I need to timebox this work and move on with other things so for now I think it's reasonable.